src/driver/rustc.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
```

Comment:
> Can’t it live in src/librustc_driver?

No because `librustdoc` depends on `librustc_driver`, and putting the `[[bin]]` sections into them will make Cargo generate two `librustc_driver`s: one in `<target-dir>/deps` with metadata, one in `<target-dir>` without metadata. This will be problematic for linking later.